README.md

```diff
@@ -106,6 +120,18 @@ in your terminal. The project will then be compiled and tested before finally be
 
 We recommend enabling annotation processing in your IDE https://immutables.github.io/apt.html.
 
+## Sinch REST SMS API 1.1.6 to 1.1.7 Migration guide
+
+In version 1.1.7 MMS messages support was added  into the library, which will require code changes if you want to upgrade.
```

Comment:
This will be 2.0.0

Reply:
Aha, ok, will update. Just used the version we have in latest local release